test/CMakeLists.txt

```diff
@@ -3,11 +3,21 @@
 # By default, we run the spec tests only if python3 is available.
 # To require the spec tests, compile with -DSPEC_TESTS=1
 
-if (SPEC_TESTS)
-  find_package(PythonInterp 3 REQUIRED)
-else(SPEC_TESTS)
-  find_package(PythonInterp 3)
-endif(SPEC_TESTS)
+if(SPEC_TESTS)
+  set(PYTHON_REQUIRED REQUIRED)
+else()
+  set(PYTHON_REQUIRED)
+endif()
+
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.12)
+  find_package(Python3 ${PYTHON_REQUIRED} COMPONENTS Interperter)
```

Comment:
`Interperter` looks like a typo.

Reply:
Oh, I see it was fixed in a later commit.

Reply:
Yeah; I want to finish up some more cleanups and then see if we can add some additional coverage through GHA.